packages/dai-plugin-mcd/src/CdpType.js

```diff
@@ -92,34 +80,34 @@ export default class CdpType {
       this._getSpotInfo(),
       this._getPar()
     ]);
-    return this.getPriceSync();
+    return this.price;
   }
 
-  getPriceSync() {
-    return price(
+  get price() {
+    return math.price(
       this.currency,
       this._getCached('par'),
       this._getCached('vatInfo').spot,
-      this.getLiquidationRatioSync()
+      this.liquidationRatio
     );
   }
 
   async getLiquidationPenalty() {
     await this._getCatInfo();
-    return this.getLiquidationPenaltySync();
+    return this.liquidationPenalty;
   }
 
-  getLiquidationPenaltySync() {
-    return liquidationPenalty(this._getCached('catInfo').chop);
+  get liquidationPenalty() {
+    return math.liquidationPenalty(this._getCached('catInfo').chop);
   }
 
   async getAnnualStabilityFee() {
     await this._getJugInfo();
-    return this.getAnnualStabilityFeeSync();
+    return this.annualStabilityFee;
   }
 
-  getAnnualStabilityFeeSync() {
-    return annualStabilityFee(this._getCached('jugInfo').duty);
+  get annualStabilityFee() {
```

Comment:
what do you think about marking these getters as private (i.e. prefixing with underscore :|) ? It seems to me we're not expected to call them directly, since it won't work without the cache.

Reply:
oh, these are definitely intended to be called directly. in its final form, we would guarantee that the cache would be populated by the time you have the instance. but i wanted to get some eyes on this even without that guarantee, since it would still be useful for the CDP portal.